state-storage/lib/state-storage.test.ts

```diff
@@ -1,143 +1,176 @@
 import {StateStorage} from "./state-storage";
-
-export class FakeAwsSuccessResult {
-    constructor(private value: any) {}
-
-    promise() {
-        return Promise.resolve(this.value);
-    }
-}
+import {SSMClient} from "@aws-sdk/client-ssm";
 
 describe('StateStorage', () => {
+    const mockSSMClient = {
+        send: jest.fn()
+    };
+    let sut;
+
     describe('#set', () => {
-        let sut = new StateStorage('/prefix/');
+        beforeEach(() => {
+            mockSSMClient.send.mockReset();
+            sut = new StateStorage('/prefix/', mockSSMClient as any as SSMClient);
```

Comment:
Consider using unknown instead of any

Reply:
Thank you. I couldn't remember if it was unknown, never, or what in the moment and took the lazy approach. Practically speaking, it makes no difference since the `any` variant never goes into a variable, but `unknown` is the best choice. I have the change pending, waiting for other feedback.